src/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -50,8 +50,20 @@ public function get_properties() {
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
+			'quantity_min'       => [
+				'description' => __( 'The minimum quantity of this item in the cart.', 'woo-gutenberg-products-block' ),
+				'type'        => 'integer',
+				'context'     => [ 'view', 'edit' ],
+				'readonly'    => true,
+			],
 			'quantity_limit'       => [
-				'description' => __( 'The maximum quantity than can be added to the cart at once.', 'woo-gutenberg-products-block' ),
+				'description' => __( 'The maximum quantity that can be added to the cart at once.', 'woo-gutenberg-products-block' ),
+				'type'        => 'integer',
+				'context'     => [ 'view', 'edit' ],
+				'readonly'    => true,
+			],
+			'quantity_step'       => [
+				'description' => __( 'The quantity step of this item in the cart.', 'woo-gutenberg-products-block' ),
```

Comment:
I can't really give an honest review if this makes sense or not, cc @mikejolley do you feel like adding those two fields make sense here?

Use cases for this are plugins that has bundles or purchase limitation.

Such limitation would be created on the server, currently, a call would be made but the server would return a notice.

Reply:
Not quite sure if these need to be part of the schema. If not, then we'll probably need to filter the values fed into the `QuantitySelector` component from `CartLineItemRow`.

Reply:
> Such limitation would be created on the server, currently, a call would be made but the server would return a notice.

That's why I initially thought that quantity input field attributes like this should be part of the API -- to let the API validate the values. I'm not sure if that's sound thinking now that I'm looking at it again.

If this stays out of the API, how should third parties validate quantity changes?